testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/ExtractJsonResponseAction.scala

```diff
@@ -137,4 +147,45 @@ object ExtractJsonResponseAction extends ResponseActions {
     ContentValidator.validateIntegerString(listIndex, s"ExtractJson.$StringFromList.listIndex")
   }
 
+  /**
+   * This method validates the parameters of the StringFromJsonPath type of response action.
+   *
+   * @param assertion The ResponseAction instance containing the response action details.
+   */
+  private def validateStringFromJsonPath(assertion: ResponseAction): Unit = {
+    val cacheKey = assertion.params.getOrElse("cacheKey", throw new IllegalArgumentException(s"Missing required 'cacheKey' parameter for extract $StringFromJsonPath logic"))
+    val jsonPath = assertion.params.getOrElse("jsonPath", throw new IllegalArgumentException(s"Missing required 'jsonPath' parameter for extract $StringFromJsonPath logic"))
+    val cacheLevel = assertion.params.getOrElse("cacheLevel", throw new IllegalArgumentException(s"Missing required 'cacheLevel' parameter for extract $StringFromJsonPath logic"))
+
+    ContentValidator.validateNonEmptyString(jsonPath, s"ExtractJson.$StringFromJsonPath.jsonPath")
+    ContentValidator.validateNonEmptyString(cacheKey, s"ExtractJson.$StringFromJsonPath.cacheKey")
+    ContentValidator.validateNonEmptyString(cacheLevel, s"ExtractJson.$StringFromJsonPath.cacheLevel")
+  }
+
+  /**
+   * This method extracts a string from a JSON array response at a given json path
+   * and stores it in a runtime cache with a given key and expiration level.
+   *
+   * @param response          The Response instance containing the JSON body.
+   * @param jsonPath          The json path in the JSON from which to extract the string.
+   * @param cacheKey          The key to use when storing the extracted string in the runtime cache.
+   * @param runtimeCacheLevel The expiration level to use when storing the extracted string in the runtime cache.
+   * @return A Try[Unit] indicating whether the string extraction and caching operation was successful or not.
+   */
+  private def stringFromJsonPath(response: Response, jsonPath: String, cacheKey: String, runtimeCacheLevel: String): Try[Unit] = Try {
+    val configuration = Configuration.defaultConfiguration().addOptions(com.jayway.jsonpath.Option.SUPPRESS_EXCEPTIONS)
```

Comment:
```suggestion
    import com.jayway.jsonpath.{Option => JsonPathOption} // this rename can be at the top of the file, too
    val configuration = Configuration.defaultConfiguration().addOptions(JsonPathOption.SUPPRESS_EXCEPTIONS)
```

Reply:
It looks better like that. 
Fixed in commit 66e35c3c68c28063ee6086a1f587c0063a5a6929.